stubs/opencv-python/cv2/cv2.pyi

```diff
@@ -0,0 +1,4551 @@
+    def getTopK(self, *args, **kwargs): ...  # incomplete
+    def setInputSize(self, input_size) -> None: ...
+    def setNMSThreshold(self, nms_threshold) -> None: ...
+    def setScoreThreshold(self, score_threshold) -> None: ...
+    def setTopK(self, top_k) -> None: ...
+
+class FaceRecognizerSF:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def alignCrop(self, *args, **kwargs): ...  # incomplete
+    def create(self, *args, **kwargs): ...  # incomplete
+    def feature(self, *args, **kwargs): ...  # incomplete
+    def match(self, *args, **kwargs): ...  # incomplete
+
+class FarnebackOpticalFlow(DenseOpticalFlow):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def create(self, *args, **kwargs): ...  # incomplete
+    def getFastPyramids(self, *args, **kwargs): ...  # incomplete
+    def getFlags(self, *args, **kwargs): ...  # incomplete
+    def getNumIters(self, *args, **kwargs): ...  # incomplete
+    def getNumLevels(self, *args, **kwargs): ...  # incomplete
+    def getPolyN(self, *args, **kwargs): ...  # incomplete
+    def getPolySigma(self, *args, **kwargs): ...  # incomplete
+    def getPyrScale(self, *args, **kwargs): ...  # incomplete
+    def getWinSize(self, *args, **kwargs): ...  # incomplete
+    def setFastPyramids(self, fastPyramids) -> None: ...
+    def setFlags(self, flags) -> None: ...
+    def setNumIters(self, numIters) -> None: ...
+    def setNumLevels(self, numLevels) -> None: ...
+    def setPolyN(self, polyN) -> None: ...
+    def setPolySigma(self, polySigma) -> None: ...
+    def setPyrScale(self, pyrScale) -> None: ...
+    def setWinSize(self, winSize) -> None: ...
+
+class FastFeatureDetector(Feature2D):
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def create(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    def getNonmaxSuppression(self, *args, **kwargs): ...  # incomplete
+    def getThreshold(self, *args, **kwargs): ...  # incomplete
+    def getType(self, *args, **kwargs): ...  # incomplete
+    def setNonmaxSuppression(self, f) -> None: ...
+    def setThreshold(self, threshold) -> None: ...
+    def setType(self, type) -> None: ...
+
+class Feature2D:
+    def __init__(self, *args, **kwargs) -> None: ...  # incomplete
+    def compute(self, *args, **kwargs): ...  # incomplete
+    def defaultNorm(self, *args, **kwargs): ...  # incomplete
+    def descriptorSize(self, *args, **kwargs): ...  # incomplete
+    def descriptorType(self, *args, **kwargs): ...  # incomplete
+    def detect(self, *args, **kwargs): ...  # incomplete
+    def detectAndCompute(self, *args, **kwargs): ...  # incomplete
+    def empty(self, *args, **kwargs): ...  # incomplete
+    def getDefaultName(self, *args, **kwargs): ...  # incomplete
+    @overload
+    def read(self, fileName) -> None: ...
+    @overload
+    def read(arg1) -> None: ...
```

Comment:
This one might also need a self arg, but...
```
In [18]: f = cv2.Feature2D()

In [19]: f.read??
Docstring:
read(fileName) -> None
.   



read(arg1) -> None
.
Type:      builtin_function_or_method

In [20]: f.read("x")
zsh: segmentation fault  python -m IPython
```


If you pass an int it tells you what it expects

```
In [4]: f.read(1)
---------------------------------------------------------------------------
error                                     Traceback (most recent call last)
<ipython-input-4-5a354f952aa4> in <module>
----> 1 f.read(1)

error: OpenCV(4.5.5) :-1: error: (-5:Bad argument) in function 'read'
> Overload resolution failed:
>  - Can't convert object to 'str' for 'fileName'
>  - Expected cv::FileNode for argument 'arg1'
```